libraries/ms-drivers/inc/pcf8523_rtc_defs.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+typedef enum {
+  CR1 = 0,
+  CR2,
+  CR3,
+  NUM_CONTROL_REG,
+} Pcf8523ControlReg;
+
+typedef enum {
+  SECONDS = 3,
+  MINUTES,
+  HOURS,
+  DAYS,
+  WEEKDAYS,
+  MONTHS,
+  YEARS,
+  NUM_TIME_REG = 7,
+} Pcf8523TimeReg;
+
+typedef enum {
+  MINUTE_ALARM = 10,
+  HOUR_ALARM,
+  DAY_ALARM,
+  WEEKDAY_ALARM,
+  NUM_ALARM_REG = 4,
+} Pcf8523AlarmReg;
+
+typedef enum {
+  OFFSET = 14,
+  NUM_OFFSET_REG = 1,
+} Pcf8523OffsetReg;
+
+typedef enum {
+  CIE = 0,
+  AIE,
+  SIE,
+  TIME_12_24,
+  SR,
+  STOP,
+  T,
+  CAP_SEL,
+} Control1;
+
+typedef enum {
+  CTBIE = 0,
+  CTAIE,
+  WTAIE,
+  AF,
+  SF,
+  CTBF,
+  CTAF,
+  WTAF,
+} Control2;
+
+typedef enum {
+  BLIE = 0,
+  BSIE,
+  BLF,
+  BSF,
+  PM,
+} Control3;

```

Comment:
You might want to add prefixes on some of these names - `T` is a bit of an ambiguous identifier but `PCF8523_CONTROL_1_T` is less ambiguous. Also you're missing `NUM_` members at the end of the control enums.